docs/en/rst/installing/mysql.rst

```diff
@@ -3,13 +3,12 @@
 MySQL
 #####
 
-It is strongly advised to use MariaDB which is a drop-in replacement for
-MySQL and is fully compatible with Bugzilla.  If MySQL must be used, be
-aware that the minimum required version is MySQL 5.0.15 and that MySQL 8
-or higher cannot be used at this time.
+It is strongly advised to use MariaDB which is fully compatible with Bugzilla.  
+If MySQL must be used, be aware that the minimum required version is 
+MySQL 5.6.12.
```

Comment:
I also think the "strongly-advised" at the top isn't necessary anymore, that was leftover from when it didn't work with MySQL 8+.  Probably Just remove this paragraph from both files, and lead off both of them with "The minimum required version is..."